src/components/permission/PermissionSpec.tsx

```diff
@@ -0,0 +1,140 @@
+import { CommandData, PermissionData } from "@site/src/components/permission/Permissions";
+
+export const vanillaCommands: CommandData = {
+  "0": "minecraft.command",
+  "1": [
+    ["advancement", [], 2],
```

Comment:
I am not really enjoying this data structure, can't we just use objects?

Reply:
It's the data structure used before in the vanilla command permissions thing. I just copied it.

Reply:
see convo on discord, out of scope for this PR but if nobody else wants to I volunteer to migrate that into proper fields 
![image](https://github.com/PaperMC/docs/assets/2185527/61d5c0c7-97b3-497d-9d35-3d43b1161bfa)